src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/CompletionListCache.cs

```diff
@@ -3,68 +3,71 @@
 
 using System;
 using System.Collections.Generic;
-using Microsoft.CodeAnalysis.Razor.Completion;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.Completion
 {
     internal sealed class CompletionListCache
     {
         // Internal for testing
-        internal static readonly int MaxCacheSize = 3;
+        internal static readonly int MaxCacheSize = 10;
```

Comment:
Expanded the cache size because when this starts storing delegated completion lists there will be 2x as many completion lists (delegated and Razor). Trippled it for now because there's limited drawback in doing so.